interface-definitions/include/firewall/common-rule.xml.i

```diff
@@ -95,6 +95,51 @@
     </constraint>
   </properties>
 </leafNode>
+<node name="ct-status">
+  <properties>
+    <help>Connection status in conntrack</help>
+  </properties>
+  <children>
+    <leafNode name="dnat">
+      <properties>
+        <help>Set when connection needs DNAT in original direction</help>
+        <completionHelp>
+          <list>enable disable</list>
```

Comment:
I understand that you added "enable/disable" for uniformity with the `state` node, so I can't blame you. But our general agreement was that boolean flags should be made valueless (i.e. just `set ct-status dnat`), since values like "enable/disable" or "true/false" don't add any information to the config. They also make code more complicated since you need to handle the case when a node is absent and also the case when it's set to "disable".

I believe we should apply that policy here and make it valueless, since eventually the `state` node will also be converted to valueless. A little temporary non-uniformity is an acceptable cost for less migration in the future.